code/game/gamemodes/blob/theblob.dm

```diff
@@ -148,9 +148,7 @@ var/list/blob_looks
 
 /obj/effect/blob/beam_disconnect(var/obj/effect/beam/B)
 	..()
-	apply_beam_damage(B)
```

Comment:
For some reason, I thought this had the bonus of preventing an exploit to get extra damage on blobs.

Coding while tired is probably a bad idea. :thinking: